src/main/kotlin/com/project/goms/domain/studentCouncil/presentation/StudentCouncilController.kt

```diff
@@ -61,4 +63,10 @@ class StudentCouncilController(
             .let { studentCouncilConverter.toResponse(it) }
             .let { ResponseEntity.ok(it) }
 
+    @PostMapping("/ban")
```

Comment:
이렇게 표시했을때 클라이언트는 /api/v1/student-council/ban 으로 요청할텐데 학생들을 ban 하는건지 외출을 ban 하는건지 이 api 하나만 보고는 의미를 단번에 파악하기는 어려울거 같습니다. 

```suggestion
    @PostMapping("/outing/ban")
```
위코드와 같이 외출을 ban 시킨다는 api로 의미를 단번에 파악하게 하면 좋을거 같습니다.